python/tk_framework_desktopserver/server.py

```diff
@@ -21,13 +21,15 @@
 
 from .errors import MissingCertificateError, PortBusyError
 
+from .logger import get_logger
+
 
 class Server:
```

Comment:
We should, it simply slipped through everyone's radar. Thanks for pointing it out.